### DISCUSSION TITLE
[Bug] Wrong Discord message on start after automatic_update

### DISCUSSION THREAD

**Discussion by AlexDiwong** [mbround18/valheim-docker#247] [Help!]
The discord webhook sends two Stop messages and no start message after automatic restart due to automatic_update=1.
The second message says "Stop Successful" which  leads me to believe it is supposed to say "Start Successful".
<img width="409" alt="Screenshot 2021-03-04 at 09 04 45" src="https://user-images.githubusercontent.com/3648302/109931204-d728a280-7cc8-11eb-8dd7-bf182d33336c.png">

**Comment by CosmicHorrorDev** [comment]
Moving this over to discussions with the other questions/bugs (#61)

**Comment by CosmicHorrorDev** [comment]
Are you on a recent pull of the image? I'm not seeing this same behavior after building a fresh image from the repo. Here are the notifications from the cron job running

![image](https://user-images.githubusercontent.com/30302768/109939113-bba8a200-7c9e-11eb-85e6-2835a2e96dbc.png)

**Reply by AlexDiwong** [reply]
I set this up yesterday evening, which I guess should have pulled the latest version right?

**Reply by AlexDiwong** [reply]
Also I guess this line isn't working properly since it always sends "Your Valheim Server" 
https://github.com/mbround18/valheim-docker/blob/main/src/notifications/discord.rs#L49

**Reply by AlexDiwong** [reply]
I might also add that I have this chat behaviour for two separate servers set up independently from each other.

**Reply by TheDakk** [reply]
@LovecraftianHorror I posted this yesterday in another thread and attached the log:

https://github.com/mbround18/valheim-docker/discussions/228#discussioncomment-425509

**Reply by ericbarrette** [reply]
I also had the same issue this morning during the auto-update schedule.  The log shows the two notifications been sent out for stopping the server, but there is no trace in the log trying to send the `start` notification.

I pulled the image for the first time yesterday around noon (EST), but I see there is an updated image this morning, so I'll pull it and see if the same thing happens.

Worth mentioning the server was still restarted properly.

**Reply by CosmicHorrorDev** [reply]
> Also I guess this line isn't working properly since it always sends "Your Valheim Server" 
> https://github.com/mbround18/valheim-docker/blob/main/src/notifications/discord.rs#L49

Ah I see the problem causing that, thanks for pointing it out!

As for the notifications not all sending I believe @mbround18 is looking into it currently. I didn't have any issues when testing, but I was building my image from`main`

**Reply by AlexDiwong** [reply]
Great! Thanks so much for the quick response, I really appreciate what you guys are doing here! 💥
